doc/stream-filter.md

```diff
@@ -30,7 +30,11 @@ predicate `pred` is any operation, that given a value `x` of type `T`, makes the
 following valid:
 
 ~~~{.cpp}
```

Comment:
Remove lines with comments and leave the example as

~~~c++
if (predicate(item)) { /* ... */ }
if (!predicate(item)) { /* ... */ }
~~~